src/Xamarin.Android.Build.Tasks/Xamarin.Android.Tooling.targets

```diff
@@ -53,15 +52,18 @@ projects.
   <Target Name="_ResolveSdks" DependsOnTargets="$(_ResolveSdksDependsOnTargets)">
     <PropertyGroup>
       <_AndroidAllowMissingSdkTooling Condition=" '$(_AndroidAllowMissingSdkTooling)' == '' ">False</_AndroidAllowMissingSdkTooling>
-      <_ResolveSdksDirectories Condition=" '$(UsingAndroidNETSdk)' == 'True' ">@(ResolvedFrameworkReference->'%(TargetingPackPath)\data\')</_ResolveSdksDirectories>
-      <_ResolveSdksDirectories Condition=" '$(UsingAndroidNETSdk)' != 'True' ">$(_XATargetFrameworkDirectories)</_ResolveSdksDirectories>
+      <!-- When using .NET 5, provide a list of paths to all framework reference directories, e.g.
+            `packages\xamarin.android.app.ref\10.3.99.160\ref\netcoreapp5.0\` and
+            `C:\Program Files\dotnet\packs\Microsoft.NETCore.App.Ref\5.0.0-preview.2.20160.6\ref\netcoreapp5.0\`
+      -->
+      <_XATargetFrameworkDirectories Condition=" '$(UsingAndroidNETSdk)' == 'True' ">@(ResolvedFrameworkReference->'%(TargetingPackPath)\ref\$(TargetFramework)')</_XATargetFrameworkDirectories>
```

Comment:
If anyone is wondering where `@(ResolvedFrameworkReference)` comes from:

![image](https://user-images.githubusercontent.com/840039/78599695-38c16200-7817-11ea-84d4-0faeb42defb5.png)

I wish there was something better than doing `'%(TargetingPackPath)\ref\$(TargetFramework)'`, but that seems like all we have.

Reply:
I was thinking about this too, we could do something like this instead...

```xml
<_XATargetFrameworkDirectories Include="@(ReferencePath->'%(RootDir)%(Directory)')" Condition="'%(FileName)%(Extension)' == 'Mono.Android.dll'" />
```

Maybe `'%(TargetingPackPath)\ref\$(TargetFramework)'` is good enough for now though.